specification/mediaservices/resource-manager/Microsoft.Media/stable/2020-05-01/streamingservice.json

```diff
@@ -1549,21 +1768,21 @@
       "properties": {
         "description": {
           "type": "string",
-          "description": "The StreamingEndpoint description."
+          "description": "The streaming endpoint description."
         },
         "scaleUnits": {
           "type": "integer",
           "format": "int32",
-          "description": "The number of scale units.  Use the Scale operation to adjust this value."
+          "description": "The number of scale units. Use the Scale operation to adjust this value."
```

Comment:
Should we link to the article on quota and also details on scaling here?

Reply:
I will add the link once we fix that doc page. We don't link quotas and limits for any other types of objects.